### DISCUSSION TITLE
IBANs of China, Afghanistan and Russia

### DISCUSSION THREAD

**Discussion by guillermosanchezg** [mdomke/schwifty#56] [Q&A]
Morning,

I need check the format and also from where are the IBANs that I get but as far as I can see I cannot do anything to work with the countries in the tittle.

Any idea or solution?

Thanks.

**Comment by mdomke** [comment]
As to my knowledge, neither China nor Afghanistan or Russia have registered a national IBAN format with SWIFT. Do you have an example for IBANs from those countries and maybe a specification of the format?

**Reply by guillermosanchezg** [reply]
As far as I could research I found nothing. That's why I came here to ask. But it is possible that they do not exist.
If you know anything about it, please let me know. 
Thank you.
